### ISSUE TITLE
http doesn't work

### ISSUE THREAD

**Issue by miroswan** [savagegus/ansible-consul#158]
https://github.com/savagegus/ansible-consul/blob/master/templates/consul.json.j2#L58-L60

I believe you'll need to set http to the client address in order for this to work?

**Comment by mhamann** [comment]
Yep, I ran into the same issue. Couldn't hit UI or get other things to talk to consul other than the main RPC.

Unfortunately, this will also require some changes to dnsmasq config (if used), especially in conjunction with Docker.
